packages/databricks-vscode/src/extension.ts

```diff
@@ -126,6 +126,33 @@ export async function activate(
         )
     );
 
+    // manage contexts for experimental features
+    function updateFeatureContexts() {
+        commands.executeCommand(
```

Comment:
We have a `CustomWhenContext` object that allows manipulation of the context. 
https://github.com/databricks/databricks-vscode/pull/770/files#diff-6cb0770ff84b4a7174d5ef92c2d1e64dac98beb798d5bc4cbf4df2ce7b423f0dL53

Reply:
Perfect. Moved the code there.